app/controllers/audits_controller.rb

```diff
@@ -59,8 +65,34 @@ def content_items
     @content_items ||= search.content_items.decorate
   end
 
-  def next_item
-    @next_item ||= content_items.next_item(content_item)
+  def compute_next_item_and_page
+    # Get a list of ContentItem ids for everything on the current
```

Comment:
This method has two side effects, the `@next_item`, but also the `@next_link_filter_params`, which is like the `filter_params`, but can include a different page.

I did think of having some function that would return the two bits of information, e.g. `next_item, next_page = next_item_and_page(scope, current_content_item)`, but decided against in as just having a method that set instance variables seemed similar to the style of the AuditsController.

I'm not against having it in a class, I put it in the controller for a start to keep the complexity low.